test/jdk/java/awt/FullScreen/FullScreenInsets/FullScreenInsets.java

```diff
@@ -21,34 +21,41 @@
  * questions.
  */
 
+import javax.imageio.ImageIO;
 import java.awt.AWTException;
 import java.awt.Color;
 import java.awt.Dimension;
-import java.awt.DisplayMode;
 import java.awt.Frame;
 import java.awt.GraphicsDevice;
 import java.awt.GraphicsEnvironment;
 import java.awt.Insets;
+import java.awt.Rectangle;
 import java.awt.Robot;
 import java.awt.Window;
 import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
 
-/**
+/*
  * @test
  * @key headful
  * @bug 8003173 7019055
  * @summary Full-screen windows should have the proper insets.
- * @author Sergey Bylokhov
+ * @requires (os.family != "linux")
```

Comment:
You should not exclude the test on the platform which has a bug in it. On Linux, this test fails if a non-accelerated java2d pipeline is used, and we just "maximize the frame", this problem is similar to Windows described in JDK-7019055.

Note that you deleted the reference to this bug https://bugs.openjdk.org/browse/JDK-8266245 from problem list, take a look at the images attached to that report, both of them have black/empty rectangle on the top. This looks similar to the initial bug this test was created https://bugs.openjdk.org/browse/JDK-8003173

So looks like this is not a test issue on all platforms, other than the  `device.getDefaultConfiguration().getBounds()` usage.

Reply:
@mrserb Since the current test fix works on other platforms (windows, mac) other than linux, can the other platforms be removed and the test be problemlisted only on linux platform until the product bug is fixed on linux then? Additionally the test was failing on Mac-ARM due to color mismatch that was fixed by adding tolerance.

Reply:
It is still unclear why tolerance is needed in this test and is not needed in the CheckCommonColors?

Reply:
On Windows, it does not work as well if a non-accelerated java2d pipeline is selected. You can test that by using this option:
`-Dsun.java2d.d3d=false`

Reply:
> On Windows, it does not work as well if a non-accelerated java2d pipeline is selected. You can test that by using this option: `-Dsun.java2d.d3d=false`

Or rather try with `-Dsun.java2d.d3d=True` and check if Direct3D is enabled. If it isn't, you won't see the difference.